addons/binding/pom.xml

```diff
@@ -31,6 +31,7 @@
     <module>org.openhab.binding.feed.test</module>
     <module>org.openhab.binding.folding</module>
     <module>org.openhab.binding.freebox</module>
+    <module>org.openhab.binding.gpio</module>
```

Comment:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).